mutacc/builds/build_case.py

```diff
@@ -69,7 +69,7 @@ def _build_variants(self, padding=None, vcf_parse=None):
         variant_objects = []
 
         for variant_object in get_variants(
-            self.input_case["variants"], padding=padding, vcf_parse=vcf_parse
+            self.input_case["variants"], padding=padding, sv_padding=sv_padding, vcf_parse=vcf_parse
```

Comment:
Consider keywording the first param for the function as well

Reply:
will do!